File: main/scala/mod/iceandshadow3/util/SMath.java
package mod.iceandshadow3.util;

public class SMath {
	public static int bound(int min, int what, int max) {
		return Math.min(max, Math.max(min, what));
	}
	public static long bound(long min, long what, long max) {
		return Math.min(max, Math.max(min, what));
	}
}

File: main/scala/mod/iceandshadow3/util/SMath.scala
package mod.iceandshadow3.util

object SMath {
  def bound(min: Int, what: Int, max: Int) = Math.min(max, Math.max(min, what))
  def bound(min: Long, what: Long, max: Long) = Math.min(max, Math.max(min, what))
}